source/documentation/other-topics/migrate-to-live.html.md.erb

```diff
@@ -46,22 +46,26 @@ During the process of migration high velocity apps will be unable to deploy to "
 
 ## Step 2 - Amend your ingress resource
 
-To ensure that you can send traffic to both clusters simultaneously you must add an annotation to your `ingress` resource to allow traffic weighting.
+Amend your ingress resource of "Live-1"
 
-This looks as follows:
+  To ensure that you can send traffic to both clusters simultaneously you must add an annotation to your `ingress` resource to allow traffic weighting.
 
-```yaml
-apiVersion: networking.k8s.io/v1beta1
-kind: Ingress
-metadata:
-  annotations:
-    external-dns.alpha.kubernetes.io/set-identifier: <ingress-name>-<environment-name>-<colour>
-    external-dns.alpha.kubernetes.io/aws-weight: "50"
-```
+  The annotation looks as follows:
 
-(Note - Please change the `ingress-name` and `environment-name` values in the above example, you can get the `environment-name` value from your namespace label "cloud-platform.justice.gov.uk/environment-name". The `colour` should be `blue` for ingress in live-1 and `green` for ingress in EKS `live` cluster)
+  ```yaml
+  apiVersion: networking.k8s.io/v1beta1
+  kind: Ingress
+  metadata:
+    annotations:
+      external-dns.alpha.kubernetes.io/set-identifier: <ingress-name>-<namespace-name>-<colour>
+      external-dns.alpha.kubernetes.io/aws-weight: "100"
+  ```
 
-The idea behind the annotation is to control where the traffic is going. Using the above example 50% of the traffic will be sent to the ingress in the "Live-1" cluster. In a later step, we will create another resource in the "Live" cluster that will weight the traffic between the two clusters.
+  (Note - Please change the `ingress-name` and `namespace-name` values in the above example.
+  The `colour` should be <span style="color:blue">**blue**</span> for ingress in "Live-1")
```

Comment:
How about just putting `-blue` in the yaml directly, rather than asking the user to do it?

Reply:
I thought it will make it clear that it needs changing for "live". But, I think it will make one less change for the user. I will amend